src/stk/molecular/topology_graphs/polymer/linear/linear.py

```diff
@@ -578,7 +581,11 @@ def _normalize_repeating_unit(repeating_unit):
         base = ord('A')
         return tuple(ord(letter)-base for letter in repeating_unit)
 
-    def _get_building_block_vertices(self, building_blocks, vertices):
+    def _get_building_block_vertices(
+        self,
+        building_blocks: dict[int, BuildingBlock],
+        vertices: Iterable[Vertex],
+    ) -> dict[BuildingBlock, type[Vertex]]:
```

Comment:
the

```python
type[Vertex]
```

annotation is wrong I think, what are you trying to express?

Reply:
Correct, is wrong. It is a Vertex.